src/components/customerQuotes/customerQuotes.js

```diff
@@ -164,8 +164,8 @@ class BaseCustomerQuotes extends React.Component {
     super(props)
     this.state = {
       index: 0,
-      left: 'https://d2lknnt52h7uhg.cloudfront.net/roa-canon/image/upload/v1548872934/web/CONFETTI_left_single.gif',
-      right: 'https://d2lknnt52h7uhg.cloudfront.net/roa-canon/image/upload/v1548872934/web/CONFETTI_right_single.gif',
+      left: 'https://res.cloudinary.com/roa-canon/image/upload/v1563820550/web/CONFETTI_static_left.png',
+      right: 'https://res.cloudinary.com/roa-canon/image/upload/v1563820550/web/CONFETTI_static_right.png',
```

Comment:
Just wondering if these should still be in state since they don't change, but only if it doesn't complicate it more.

Reply:
I like this idea, will experiment. The file seems a little out of the ordinary, so if it ends up being risky we'll push as it is. 👍